javascript-if/main.js

```diff
@@ -0,0 +1,108 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+// function isUnderFive(number) {
+//   if (number < 5) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+function isUnderFive(number) {
+  return number < 5;
+}
+
+// console.log('value of isUnderFive(4): ', isUnderFive(4));
+// console.log('value of isUnderFive(10): ', isUnderFive(10));
+// console.log('value of isUnderFive(5): ', isUnderFive(5));
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+// console.log('value of isEven(4): ', isEven(4));
+// console.log('value of isEven(10): ', isEven(10));
+// console.log('value of isEven(5): ', isEven(5));
+
+function startsWithJ(string) {
+  return string[0] === 'J';
+}
+
+// console.log('value of startsWithJ('JavaScript'): ', startsWithJ('JavaScript'));
+// console.log('value of startsWithJ('PHP'): ', startsWithJ('PHP'));
+// console.log('value of startsWithJ('HTML'): ', startsWithJ('HTML'));
+// console.log('value of startsWithJ('Java'): ', startsWithJ('Java'));
+// console.log('value of startsWithJ('Kotlin'): ', startsWithJ('Kotlin'));
+// console.log('value of startsWithJ('C#'): ', startsWithJ('C#'));
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+
+// var bart = {
+//   name: 'Bartholomew JoJo Simpson',
+//   age: 10
+// };
+
+// console.log('value of isOldEnoughToDrink(bart): ', isOldEnoughToDrink(bart));
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 18;
+}
```

Comment:
Still not the right minimum driving age:

---

![image](https://user-images.githubusercontent.com/16143771/145284672-41644692-1729-40cd-bab8-80f17e50bc8e.png)